### PR TITLE
add new team member

### PR DESCRIPTION
**Pull request by Japhetb**
Add new team member profile

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -656,12 +656,23 @@ <h4>Amanda Jepson</h4>
 
         </div>
           <div class="row"> 
-            <!--new change-->
-            <!-- awesome code about me-->
-            <!--charlesmotaroki@students.uonbi.ac.ke	-->
-            <!--wanjaerica@gmail.com	-->
-            <!--mboyaivy@gmail.com-->
-            <!--machoguabednego6@gmail.com	-->
```
Comment (RuthNjeri) [inline index.html:664]: You were supposed to include your code beneath the commented line of your name and not remove anyone else's name...

File: index.html [hunk 2]
```diff
@@ -656,12 +656,23 @@ <h4>Amanda Jepson</h4>
...
+                  <div class="social">
+                    <a href=""><i class="bi bi-twitter"></i></a>
+                    <a href=""><i class="bi bi-facebook"></i></a>
+                    <a href=""><i class="bi bi-instagram"></i></a>
+                    <a href=""><i class="bi bi-linkedin"></i></a>
+                  </div>
+                </div>
+                <div class="member-info">
+                  <h4>Amanda Jepson</h4>
+                  <span>Accountant</span>
```
Comment (RuthNjeri) [inline index.html:672]: There are no links to your profile and the name and job title is not related to you... you need to fill in this section with your details.

File: index.html [hunk 3]
```diff
@@ -656,12 +656,23 @@ <h4>Amanda Jepson</h4>
...
-            <!--new change-->
-            <!-- awesome code about me-->
-            <!--charlesmotaroki@students.uonbi.ac.ke	-->
-            <!--wanjaerica@gmail.com	-->
-            <!--mboyaivy@gmail.com-->
-            <!--machoguabednego6@gmail.com	-->
+            <div class="col-lg-3 col-md-6 d-flex align-items-stretch">
+              <div class="member" data-aos="fade-up" data-aos-delay="400">
+                <div class="member-img">
+                  <img src="assets/img/team/steve.jpeg" class="img-fluid" alt="">
```
Comment (RuthNjeri) [inline index.html:662]: The picture should be of dimensions 600 pixels to 600 pixels, width and height.

**Review by RuthNjeri — CHANGES_REQUESTED** [review]

**Comment by RuthNjeri** [comment]
Hi @Japhetb, did you look at the comments above?

**Comment by Japhetb** [comment]
> 

On it

**Comment by RuthNjeri** [comment]
Hi @Japhetb , still working on this?

**Comment by RuthNjeri** [comment]
Feedback implementation took too long. Closing the PR.

**Pull request closed without merging**